docs/docs/file-system-page-creation.md

```diff
@@ -1,60 +1,39 @@
 ---
-title: Creating Pages from the File System
+title: File System Route API
 ---
 
-This page roughly documents the APIs and conventions available with the new file-based page creation features.
+This page documents the APIs and conventions available with a file system based routing API, a suite of new APIs and conventions to make the file system the primary way of creating pages. While the [`createPage`](/docs/actions/#createPage) Node API won't go away you should be able to accomplish most common tasks with this file-based API.
 
-For now, these features are marked as experimental and require a flag to utilize. Start your Gatsby site locally with the following flag:
+For now, these features are marked as experimental and require a flag to utilize. Add the following flag when using `gatsby develop` and `gatsby build`:
 
 ```shell
 GATSBY_EXPERIMENTAL_ROUTING_APIS=1 gatsby develop
 ```
 
-## Creating client-only routes
-
-Use client-only routes if you have dynamic data that does not live in Gatsby. This might be something like a user settings page, or some other dynamic content that isn't known to Gatsby at build time. In these situations, you will usually create a route with one or more dynamic segments to query data from a server in order to render your page.
-
-For example, in order to edit a user, you might want a route like `/user/:id` to fetch the data for whatever `id` is passed into the URL. You can now use square brackets (`[ ]`) in the file path to mark any dynamic segments of the URL.
-
-- `/src/pages/users/[id].js => /users/:id`
-- `/src/pages/users/[id]/group/[groupId].js => /users/:id/group/:groupId`
-
-Gatsby also supports _splat_ routes, which are routes that will match _anything_ after the splat. These are less common, but still have use cases. As an example, suppose that you are rendering images from [S3](/docs/deploying-to-s3-cloudfront/) and the URL is actually the key to the asset in AWS. Here is how you might create your file:
-
-- `/src/pages/image/[...awsKey].js => /users/*awsKey`
-- `/src/pages/image/[...].js => /app/*`
-
-Three periods `...` mark a page as a splat route. Optionally, you can name the splat as well, which has the benefit of naming the key of the property that your component receives. The dynamic segment of the file name (the part between the square brackets) will be filled in and provided to your components on a `props.params` object. For example:
+Or in the `scripts` section of your `package.json`:
 
-```js:title=/src/pages/users/[id].js
-function UserPage(props) {
-  const id = props.params.id
+```json:title=package.json
+{
+  "develop": "GATSBY_EXPERIMENTAL_ROUTING_APIS=1 gatsby develop",
+  "build": "GATSBY_EXPERIMENTAL_ROUTING_APIS=1 gatsby build"
 }
 ```
```

Comment:
do we want to help windows people out using cross-env? Like adding a note?

Reply:
Yeah, good point 👍🏻